source/_integrations/vegehub.markdown

```
@@ -0,0 +1,94 @@
+## Troubleshooting
+
+### Device is unresponsive
+
+If the device is unresponsive, wake it up using one of these methods:
+
+- Press the button on the board.
+- Disconnect and reconnect power.
+
+### Setup is failing
+
+  - Ensure the VegeHub is awake (see Device Wake-Up section)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)